jsk_fetch_robot/jsk_fetch_startup/robots/base_camera_mount.urdf.xacro

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" ?>
+<robot name="base_camera_mount" xmlns:xacro="http://www.ros.org/wiki/xacro">
+  <link name="base_camera_mount">
+    <inertial>
+      <origin rpy="0 0.523598775 0" xyz="0 0 0" />
+      <mass value="0.17" />
+      <inertia ixx="0.024" ixy="0.01" ixz="0.034" iyy="0.034" iyz="0.024" izz="0.01" />
+    </inertial>
+    <visual>
+      <origin rpy="0 0 0" xyz="0 0 0" />
+      <geometry>
+        <mesh filename="package://jsk_fetch_accessories/base_camera_mount_mk1/fetch_base_camera_mount_mk1_main.stl" scale="0.001 0.001 0.001" />
```

Comment:
do you have urdf for base_camera_mount_mk2 ? https://github.com/jsk-ros-pkg/jsk_robot/tree/master/jsk_fetch_robot/jsk_fetch_accessories/base_camera_mount_mk2
@sktometometo

Reply:
@k-okada Updated